Gemfile.lock

```diff
@@ -0,0 +1,182 @@
+GEM
```

Comment:
Why was this file added?

Reply:
My bad, I added this by mistake while merging branches